recipes/libigl/all/conanfile.py

```diff
@@ -78,15 +78,19 @@ def package_id(self):
         if self.info.options.header_only:
             self.info.clear()
 
+    def validate_build(self):
+        if (os.getenv('CONAN_CENTER_BUILD_SERVICE') is not None and
+                Version(self.version) == "2.3.0" and conan_version.major == 1 and self.settings.build_type == "Debug"):
+            raise ConanInvalidConfiguration("Debug build disabled for Conan 1.x due to excessive memory use in ConanCenter CI")
```

Comment:
I would probably remove the condition on `conan_version.major == 1` ? Ram utilisation by the build should not be affected by the version of Conan at all - perhaps this was done to be too "restrictive" since we dont build debug builds for conan 2 yet?